CMakeLists.txt

```diff
@@ -159,6 +151,11 @@ if (MSVC AND NOT ${S2_SOURCE} STREQUAL "BUNDLED")
   target_compile_options(s2::s2 INTERFACE /J)
 endif()
 
+# this might be needed since s2geometry includes it in general
+# but not for any target explicilty?
+find_package(OpenSSL REQUIRED)
+target_include_directories(s2::s2 INTERFACE ${OPENSSL_INCLUDE_DIR})
```

Comment:
This fixes the CI but would fail in the case s2geometry v0.11.0 is included via `find_package`. It should work now in all cases with [e612e2e](https://github.com/paleolimbot/s2geography/pull/22/commits/e612e2e8ab7a901e969edd0c81b51f0a14241326).